x/sentry_integration.go

```diff
@@ -177,9 +177,7 @@ func PanicHandler(out string) {
 // WrapPanics is a wrapper on panics. We use it to send sentry events about panics
 func WrapPanics() {
 	exitStatus, err := panicwrap.BasicWrap(PanicHandler)
-	if err != nil {
-		panic(err)
-	}
+	Panic(err)
```

Comment:
This is actually confusing. It looks like `panic(err)`. Can we keep the original code here?

Reply:
I reverted it. Maybe it's better to rename `x.Panic` to something like `x.Must`?

Reply:
to avoid reusing `panic`